### DISCUSSION TITLE
공공데이터 오류 해결해주세요

### DISCUSSION THREAD

**Discussion by otcroz** [SHAPE-UP/Bangida-client#38] [Q&A] [answered]
6시간 동안 해결을 못했어요..

**Comment by yellow-jam** [comment, marked as answer]
- 문제의 API...
https://www.culture.go.kr/data/openapi/openapiView.do?id=519&category=E&gubun=B#/default

- 문제의 오류...
expected begin_object but was string at line 1 column 1 path $
